makefiles/common.mk

```diff
@@ -126,3 +127,8 @@ endif
 ifneq ($(RDMA_CORE), 0)
 CXXFLAGS += -DNCCL_BUILD_RDMA_CORE=1
 endif
+
+ifdef ENABLE_PRECISION_CLIPPING_HALF
+CXXFLAGS += -DENABLE_PRECISION_CLIPPING_HALF=$(ENABLE_PRECISION_CLIPPING_HALF)
```

Comment:
Shall we remove CXXFLAGS here since ENABLE_PRECISION_CLIPPING_HALF is only for device code

Reply:
I removed the CXXFLAGS and pass the test.